src/components/RichTextEditor.tsx

```diff
@@ -118,15 +124,21 @@ export default function RichTextEditor({
 
   return (
     <Styles focus={focus} disabled={disabled}>
-      <Editor
-        disabled={disabled}
-        init={{
-          skin: false,
-          content_css: false,
-          content_style: [
-            theme.palette.mode === "dark" ? contentCssDark : contentCss,
-            theme.palette.mode === "dark" ? contentUiCssDark : contentUiCss,
-            `
+      <Suspense fallback={<></>}>
+        {theme.palette.mode === "dark" ? (
+          <RichTextEditorDarkCSS />
+        ) : (
+          <RichTextEditorLightCSS />
+        )}
+        <Editor
+          disabled={disabled}
```

Comment:
```suggestion
          key={theme.palette.mode}
          disabled={disabled}
```

Added key to re-render when theme changes.